website/common/script/cron.js

```diff
@@ -4,8 +4,10 @@
   Cron and time / day functions
   ------------------------------------------------------
  */
-import _ from 'lodash'; // eslint-disable-line lodash/import-scope
```

Comment:
I hadn't noticed this but we should really avoid importing the entire lodash in common or in client. Adding one exception defats the entire purpose of only loading individual utilities